Author: Simone Rossetto <simros85@gmail.com>
Description: fix loading of tascam firmware
 Susbtituted $DEVICE with $DEVNAME in order to make this file
 work with newer kernel.
Forwarded: no
Debian-Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=620510
Index: alsa-tools-1.0.25/usx2yloader/tascam_fpga.in
===================================================================
--- alsa-tools-1.0.25.orig/usx2yloader/tascam_fpga.in	2012-05-19 16:57:36.360424604 +0200
+++ alsa-tools-1.0.25/usx2yloader/tascam_fpga.in	2012-05-19 16:57:53.400423733 +0200
@@ -10,12 +10,12 @@
 
 if [ -x $LOADER ]; then
 	if [ -x /usr/bin/logger ]; then
-		/usr/bin/logger -t $0 "calling $LOADER for $DEVICE"
+		/usr/bin/logger -t $0 "calling $LOADER for $DEVNAME"
 	fi
 	$LOADER
 else
 	if [ -x /usr/bin/logger ]; then
-		/usr/bin/logger -t $0 "missing $LOADER for $DEVICE"
+		/usr/bin/logger -t $0 "missing $LOADER for $DEVNAME"
 	fi
 fi
 
@@ -33,12 +33,12 @@
 if [ $CONTROLLER != NOTUSED ]; then
 	if [ -x $CONTROLLER ]; then
 		if [ -x /usr/bin/logger ]; then
-			/usr/bin/logger -t $0 "starting $CONTROLLER for $DEVICE"
+			/usr/bin/logger -t $0 "starting $CONTROLLER for $DEVNAME"
 		fi
 		$CONTROLLER &
 	else
 		if [ -x /usr/bin/logger ]; then
-			/usr/bin/logger -t $0 "missing $CONTROLLER for $DEVICE"
+			/usr/bin/logger -t $0 "missing $CONTROLLER for $DEVNAME"
 		fi
 	fi
 fi
Index: alsa-tools-1.0.25/usx2yloader/tascam_fw.in
===================================================================
--- alsa-tools-1.0.25.orig/usx2yloader/tascam_fw.in	2012-05-19 16:57:36.360424604 +0200
+++ alsa-tools-1.0.25/usx2yloader/tascam_fw.in	2012-05-19 16:57:53.400423733 +0200
@@ -53,8 +53,8 @@
 fi
 
 if [ -x /usr/bin/logger ]; then
-	/usr/bin/logger -t $0 "load $FIRMWARE for $PRODUCT to $DEVICE"
+	/usr/bin/logger -t $0 "load $FIRMWARE for $PRODUCT to $DEVNAME"
 fi
-$LOADER $FLAGS -s $LOADERFIRMWARE -I $FIRMWARE
+$LOADER $FLAGS -s $LOADERFIRMWARE -I $FIRMWARE -D $DEVNAME
 
 exit 0
